app/models/company.rb

```diff
@@ -96,6 +96,37 @@ def self.create_from(row)
     new_company
   end
 
+  def size(row)
+    return row[20] if row[20] == 'Unicórnio'
```

Comment:
A empresa ser unicórnio não é excludente de ela ser Microempresa, pequena empresa, etc. Aqui você retorna se já for Unicórnio sem antes adicionar a outra classificação no array